SnailTests/ObservableTests.swift

```diff
@@ -332,4 +332,70 @@ class ObservableTests: XCTestCase {
             XCTAssert(received.last == "2")
         }
     }
+
+    func testCombineLatestNonOptional() {
+        let exp = expectation(description: "merge")
```

Comment:
should we rename "merge"?

Reply:
Absolutely thanks, copy and paste issue as you may imagine